records_mover/records/__init__.py

```diff
@@ -4,16 +4,17 @@
     'RecordsFormatType',
     'RecordsSchema',
     'RecordsFormat',
+    'DelimitedVariant',
```

Comment:
This is a new type describing the different valid variants.  It's only used by the integration tests so far, but I'd like to get more records format stuff to be mypy-friendly and this is a step.